pgjdbc/src/test/java/org/postgresql/test/jdbc3/StringTypeParameterTest.java

```diff
@@ -17,15 +22,11 @@
 import java.sql.Types;
 import java.util.Properties;
 
-public class StringTypeParameterTest extends TestCase {
+public class StringTypeParameterTest {
 
   private Connection _conn;
 
-  public StringTypeParameterTest(String name) {
-    super(name);
-  }
-
-  protected boolean setUp(String stringType) throws Exception {
+  private boolean prepare(String stringType) throws Exception {
```

Comment:
I think this behavoiur had been before I replaced `setUp` with `prepare`. Because 'setUp' had parameter, JUnit didn't pick up it (http://junit.sourceforge.net/junit3.8.1/javadoc/junit/framework/TestCase.html#setUp()).